pages/about.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/a1e854484d.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="../dist/css/style.css" />
+    <script defer src="../app/js/script.js"></script>
+    <title>Africa's Leading Conference 2023 | About Page</title>
+  </head>
+  <body class="about">
+    <header class="header">
+      <section class="header__nav-section">
+        <div class="container container-px">
+          <nav class="header__nav flex-jc-sb flex-ai-c">
+            <button id="btnHumburger" class="header__hamburger flex fle-jc-c flex-ai-c flex-collumn">
+              <span class="header__hamburger-stroke header__hamburger-top"></span>
+              <span class="header__hamburger-stroke header__hamburger-middle"></span>
+              <span class="header__hamburger-stroke header__hamburger-bottom"></span>
+            </button>
+
+            <a href="/" class="header__logo"><img src="../images/logo.svg" alt="logo" /></a>
+
+            <ul class="header__links flex-jc-c flex-ai-c">
+              <li>
+                <a href="#about-home" class="header__links-link">About</a>
+              </li>
+
+              <li>
+                <a href="#partners" class="header__links-link">sponsor</a>
+              </li>
+
+              <li><a href="#news" class="header__links-link">News</a></li>
+            </ul>
+
+            <ul class="header__menu flex flex-collumn">
+              <li class="header__menu-option header__menu-option-home">
+                <a href="/" class="header__menu-link">Home</a>
+              </li>
+              <li class="header__menu-option"><a href="#about-home" class="header__menu-link">About</a></li>
+              <li class="header__menu-option"><a href="#partners" class="header__menu-link">sponsor</a></li>
+              <li class="header__menu-option"><a href="#news" class="header__menu-link">News</a></li>
+            </ul>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/210954562-3ee802f0-b783-4276-b24e-cde62c57352f.png)
your design ⬆️ 

![image](https://user-images.githubusercontent.com/60612329/210954701-68344690-d1ca-4b2b-9e03-41230caf99d8.png)
required design ⬆️ 

- as seen about your navigation should include the section with the social media icons and other details 😄 
- your padding around your nav bar should also be reduced as well as seen above

Reply:
Got it.

Reply:
Fixed.